tools/asciidoctor/postprocessor.rb

```diff
@@ -0,0 +1,118 @@
+# Copyright (C) 2020  Matthew Glazar
+# See end of file for extended copyright information.
+
+class Copyright < Asciidoctor::Extensions::Postprocessor
+  def process document, output
+    if document.basebackend? 'html'
+      top = %{<!-- Copyright (C) 2020  Matthew Glazar -->
+<!-- See end of file for extended copyright information. -->
+}
+      bottom = %{
+
+<!--
+quick-lint-js finds bugs in JavaScript programs.
+Copyright (C) 2020  Matthew Glazar
+
+This file is part of quick-lint-js.
+
+quick-lint-js is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+quick-lint-js is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with quick-lint-js.  If not, see <https://www.gnu.org/licenses/>.
+-->
+<!--
+Other files may have different copyright and may be licensed under different
+licenses. See license.html for details.
+-->
+}
+      output = output.insert(output.index('>') + 2, top)
+      output = output.insert(output.length, bottom)
+    elsif document.basebackend? 'manpage'
+      top = %{.\\" Copyright (C) 2020  Matthew Glazar
+.\\" See end of file for extended copyright information.
+.
+.\\" Manual page for the 'man' utility.
+.
+.
+}
+      bottom = %{
+
+.\\" quick-lint-js finds bugs in JavaScript programs.
+.\\" Copyright (C) 2020  Matthew Glazar
+.\\"
+.\\" This file is part of quick-lint-js.
+.\\"
+.\\" quick-lint-js is free software: you can redistribute it and/or modify
+.\\" it under the terms of the GNU General Public License as published by
+.\\" the Free Software Foundation, either version 3 of the License, or
+.\\" (at your option) any later version.
+.\\"
+.\\" quick-lint-js is distributed in the hope that it will be useful,
+.\\" but WITHOUT ANY WARRANTY; without even the implied warranty of
+.\\" MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+.\\" GNU General Public License for more details.
+.\\"
+.\\" You should have received a copy of the GNU General Public License
+.\\" along with quick-lint-js.  If not, see <https://www.gnu.org/licenses/>.
+}
+      output = output.insert(0, top)
+      output = output.insert(output.length, bottom)
+    end
+    output
+  end 
+end
+
+class HtmlFooter < Asciidoctor::Extensions::Postprocessor
```

Comment:
Can we do something similar with the header? Specifically, I want the `<h1>` to link to the home page, and I want a paragraph under the `<h1>` which says something like "This page documents the quick-lint-js command-line interface (CLI).".

Reply:
Should be easy.

Reply:
I decided to copy the current header convention (source code link on the right).